apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/internal/flows.kt

```diff
@@ -0,0 +1,51 @@
+package com.apollographql.apollo3.internal
+
+import com.apollographql.apollo3.api.ApolloInternal
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.FlowCollector
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.flow
+
+/**
+ * This is copied from [Flow], which is marked [ExperimentalCoroutinesApi].
+ * There is a risk this API will be removed or changed in the future, which could break consumers of this library
+ * - that is why we use our own copy.
+ *
+ * TODO: remove when kotlinx.coroutines.flow.Flow.transformWhile is no longer marked ExperimentalCoroutinesApi.
+ */
+@ApolloInternal
+internal fun <T, R> Flow<T>.transformWhile(
```

Comment:
Do we need `@ApolloInternal` on something that's already `internal`?

Reply:
Nope 😅  I realized a bit later it was useless - will be gone on the next commit